examples/advanced/create-react-app/src/style-dictionary-dist/variables.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
Should we keep the style dictionary generated files not in git? I usually keep mine out of git, but other people might commit them. I don't know what the right answer is.

Reply:
If you see the other examples, I don't commit them. But in this very specific case, I needed the files in order to be able to compile the the CRA app (in case I haven't run the `build tokens` command before). Maybe the solution is just to automatically run it before the `start` command as you suggest above and we can remove the generated files from the repo.